### PR TITLE
change undefined with 0 as value to fix new Decimal error

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
Fixes [undefined value on new Decimal function #1088](https://github.com/oceanprotocol/market/issues/1088).

Changes proposed in this PR:

- fix new Decimal function error on undefined value

### CODE REVIEW

**Review by claudiaHash — APPROVED** [review]
nice fix!

File: src/@context/Pool/index.tsx [hunk 1]
```diff
@@ -197,7 +197,9 @@ function PoolProvider({ children }: { children: ReactNode }): ReactElement {
 
     // Staking bot receives half the pool shares so for display purposes
     // we can multiply by 2 as we have a hardcoded 50/50 pool weight.
-    const userPoolShares = new Decimal(poolInfoUser.poolShares)
+    const userPoolShares = new Decimal(
+      poolInfoUser.poolShares ? poolInfoUser.poolShares : 0
```
[the remark this answers is not in the document]
Comment (EnzoVezzaro) [reply src/@context/Pool/index.tsx:201]: agree 🤙🏽

[the remark this answers is not in the document]
Comment (kremalicious) [reply src/@context/Pool/index.tsx:201]: so then if we put in this change, then we can merge. Pretty sure just accepting the suggestion here on GitHub will lead to some prettier formatting error so better do it locally

[the remark this answers is not in the document]
Comment (EnzoVezzaro) [reply src/@context/Pool/index.tsx]: added manually and committed

**Review by kremalicious — APPROVED** [review]
nice, now it's a perfect 1-line-fix

**Pull request merged**